### PR TITLE
Add automatic version bumping

### PR DESCRIPTION
**Pull request by timorl**
I'm not actually sure this will work, as I'm not very familiar with GitHub actions. I also don't even know how to check it without merging. <_<"

### CODE REVIEW

File: .github/workflows/version-bump.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Autobump version
+
+on:
+  push:
+    branches:
+    - master
```
Comment (fbielejec) [inline .github/workflows/version-bump.yml:6]: it's now called main

File: .github/workflows/version-bump.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+jobs:
+  autobump:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: bump
+        run: ./bump_version.sh && git push -f origin master || echo 'No action needed.'
```
Comment (mike1729) [inline .github/workflows/version-bump.yml:15]: Should be origin main

**Review by mike1729 — CHANGES_REQUESTED** [review]

File: bump_version.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```
Comment (fbielejec) [inline bump_version.sh:1]: maybe put the script in .github/workflows or just write it plainly in the yaml file?
```
        run: |
          echo a
          echo b         
```

File: .github/workflows/version-bump.yml (hunk 2 above)
Comment (fbielejec) [inline .github/workflows/version-bump.yml:15]: there seems to be a packaged Action for commits and pushes:
https://github.com/marketplace/actions/add-commit

maybe we could just use that? Simply define it as a step after bumping the version in the Cargo file?

File: .github/workflows/version-bump.yml (hunk 1 above)
Comment (timorl) [reply .github/workflows/version-bump.yml]: Oops, fixed.

File: .github/workflows/version-bump.yml (hunk 2 above)
Comment (timorl) [reply .github/workflows/version-bump.yml]: Oops, fixed.

Comment (timorl) [reply .github/workflows/version-bump.yml]: I have seen that, but I didn't see a way of doing amends there – and creating an additional commit after every merge seems like a mess.

Comment (fbielejec) [reply .github/workflows/version-bump.yml]: O would also avoid force pushing, just in case...

File: bump_version.sh (hunk 3 above)
Comment (timorl) [reply bump_version.sh]: Added straight to the yaml file, seems somewhat strange, but I guess it should work.

File: .github/workflows/version-bump.yml (hunk 2 above)
Comment (mike1729) [reply .github/workflows/version-bump.yml]: If we amend then we have to force push. I don't know how to do a separate commit from this action.

Comment (timorl) [reply .github/workflows/version-bump.yml]: This wouldn't work with `--amend`.

**Review by fbielejec — APPROVED** [review]
if it works +1 here

**Review by mike1729 — APPROVED** [review]

**Pull request merged**